crates/astria-core/src/sequencer/v1/block/mod.rs

```diff
@@ -327,25 +337,41 @@ enum SequencerBlockErrorKind {
 /// a `[SequencerBlockHeader]`.
 #[derive(Debug)]
 pub struct SequencerBlockHeaderParts {
-    pub cometbft_header: tendermint::block::header::Header,
+    pub chain_id: tendermint::chain::Id,
+    pub height: tendermint::block::Height,
+    pub time: Time,
     pub rollup_transactions_root: [u8; 32],
-    pub rollup_ids_root: [u8; 32],
+    pub data_hash: [u8; 32],
+    pub proposer_address: account::Id,
 }
 
 #[derive(Debug, Clone, PartialEq)]
 pub struct SequencerBlockHeader {
     // the cometbft header for this sequencer block
-    cometbft_header: tendermint::block::header::Header,
+    // cometbft_header: tendermint::block::header::Header,
+    chain_id: tendermint::chain::Id,
+    height: tendermint::block::Height,
+    time: Time,
```

Comment:
If we redo these, can't we get rid of tendermint here for good? I am looking at the changed `SequencerBlockHeader` in the redefined protos, and these don't show up at the tendermint level anymore? Would much rather use native protos (or well known types).

Reply:
yeah we can - i left them as tendermint as they're originating from a tendermint chain, but can change to our own types also